src/mixins/infiniteScrolling.js

```diff
@@ -0,0 +1,37 @@
+const infiniteScrolling = {
+	data() {
+		return {
+			bottom: false,
+			scrollY: 0,
+		}
+	},
+	created() {
+		window.scrollTo(0, 0);
+		window.addEventListener("scroll", () => {
+			this.bottom = this.bottomVisible();
+			this.scrollY = window.scrollY;
+		});
+	},
+	beforeDestroy() {
+		window.removeEventListener("scroll", () => {
+			this.bottom = this.bottomVisible();
+			this.scrollY = window.scrollY;
+		});
+	},
+	methods: {
+		bottomVisible() {
```

Comment:
Please use our naming guidelines for mixins to prevent future namespace conflicts:
https://vuepress.test.schul-cloud.org/0-GettingStarted/3-DirectoryStructure.html#src-mixins
```suggestion
		$_bottomVisible() {
```